lib/Constants.ts

```diff
@@ -1,2 +1,2 @@
-export const JIRA_ISSUE_PATTERN: RegExp = /[A-Z]+-\d+/g;
+export const JIRA_ISSUE_PATTERN: RegExp = /[A-Z][A-Z0-9_]*-\d+/g;
```

Comment:
Based on [Jira Doc](https://confluence.atlassian.com/adminjiraserver/changing-the-project-key-format-938847081.html).